webapp/src/components/legal_hold_table/legal_hold_row/reset_confirmation_modal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+
+interface Props {
+    show: boolean;
+    onHide: () => void;
+    onConfirm: () => void;
+}
+
+const ResetConfirmationModal = ({show, onHide, onConfirm}: Props) => {
+    return (
+        <Modal
+            show={show}
+            onHide={onHide}
+        >
+            <Modal.Header closeButton={true}>
+                <Modal.Title>{'Are you sure?'}</Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                {'This will modify the legal hold status forcefully setting the status to \'not running\'. Only run this if you know what you are doing.'}
```

Comment:
```suggestion
                {'This will modify the legal hold status forcefully setting the status to \'not running\'. Only do this if you know the status should be \'idle\'.'}
```

I think the status should be reset when plugin is reloaded.  Also, should the status update in KVStore protected by the cluster mutex?  Any danger of race condition?

Reply:
Is it `not running` or `idle`?

Reply:
> Also, should the status update in KVStore protected by the cluster mutex? Any danger of race condition?

I see the value is set atomically.